types/nut-pipe/index.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for nut-pipe 1.1
+// Project: https://github.com/nodejs-projects-kenanhancer/nut-pipe
+// Definitions by: kenan hancer https://github.com/kenanhancer
```

Comment:
Still need the `<` brackets to make the parser happy

Reply:
Hi Andrew thanks for letting me know :) i hope it is ok this time.